### PR TITLE
[REF] Update US emission factors

### PR DESCRIPTION
**Pull request by mathilde-daugy**
NOTE: I haven't had time to review all the new EFs and will try to go through them tomorrow

## Issue

Update US emission factors with eGRID 2021 values. The process is the same as last year, see [wiki](https://github.com/electricitymaps/electricitymaps-contrib/wiki/US-emission-factors) for documentation 

Computation notebook: [US_emission_factors.ipynb](https://colab.research.google.com/drive/1_FAAuTtIdMuORZKsqHnd9vMNHlQgYarh#scrollTo=IjloJbBZzub3)

## Description
Updated yaml files for all US zones that have REF

### Preview

NA

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
We don't have any 2021 data? Would be nice to have yearly changes so they aren't as drastic perhaps?

File: config/zones/US-NW-TPWR.yaml [hunk 1]
```diff
@@ -101,9 +104,12 @@ emissionFactors:
...
     biomass:
-      datetime: '2020-01-01'
-      source: eGrid 2020
-      value: 25.02597585
+      - datetime: '2020-01-01'
+        source: eGrid 2020
+        value: 25.02597585
+      - datetime: '2022-01-01'
+        source: eGRID 2021, eGrid 2021
+        value: 277.252
```
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply config/zones/US-NW-TPWR.yaml:112]: Should this be 2021? Not 2022?

File: config/zones/US-CAL-CISO.yaml [hunk 2]
```diff
@@ -46,21 +46,33 @@ emissionFactors:
...
     gas:
-      datetime: '2021-01-01'
-      source: eGrid 2020
-      value: 403.8432591
+      - datetime: '2021-01-01'
+        source: eGrid 2020
+        value: 403.8432591
+      - datetime: '2022-01-01'
+        source: eGRID 2021
+        value: 388.81
```
Comment (VIKTORVAV99) [inline config/zones/US-CAL-CISO.yaml:68]: These use the same source but have different datetime, is this intentional?

Comment (mathilde-daugy) [reply config/zones/US-CAL-CISO.yaml]: I fixed it last night, I don't know what happened

File: config/zones/US-CAL-CISO.yaml
Comment (florianscheidl) [inline config/zones/US-CAL-CISO.yaml:1]: Biomass has 0.0 in both cases, is that reasonable?

File: config/zones/US-CAL-IID.yaml
Comment (florianscheidl) [inline config/zones/US-CAL-IID.yaml:1]: Large shift biomass emissions:
39.38869109 -> 277.252

Comment (florianscheidl) [inline config/zones/US-CAL-IID.yaml:1]: Large shift lifecycle biomass emissions:
39.38869109 -> 277.252

File: config/zones/US-MIDA-PJM.yaml [hunk 3]
```diff
@@ -80,9 +89,12 @@ emissionFactors:
...
     oil:
-      datetime: '2021-01-01'
-      source: eGrid 2020
-      value: 751.5213327
+      - datetime: '2020-01-01'
+        source: eGrid 2020
+        value: 751.5213327
+      - datetime: '2021-01-01'
+        source: eGRID 2021
+        value: 970.02
```
Comment (mathilde-daugy) [inline config/zones/US-MIDA-PJM.yaml:97]: image
We have a big increased compared to 2021 but it is still close to the average value

File: config/zones/US-CAL-LDWP.yaml
Comment (florianscheidl) [inline config/zones/US-CAL-LDWP.yaml:1]: Large shift biomass lifecycle: 
138.0968906 -> 277.252

File: config/zones/US-FLA-FMPP.yaml
Comment (florianscheidl) [inline config/zones/US-FLA-FMPP.yaml:1]: Oil LCA doubled here

File: config/zones/US-FLA-FPC.yaml
Comment (florianscheidl) [inline config/zones/US-FLA-FPC.yaml:1]: Oil direct and LCA almost halved

File: config/zones/US-HI-HA.yaml
Comment (florianscheidl) [inline config/zones/US-HI-HA.yaml:1]: Geothermal indirect and direct went up my a lot

**Review by florianscheidl — APPROVED** [review]
**Observations**:

- Large increases or decreases in biomass lifecycle emissions factors, usually to 277 (originally very small: ~1 or very large ~900) in many zones.
- Significant changes in direct and indirect oil emission factors (50% in both directions) for several zones.
- Significant changes in direct and indirect geothermal emission factors (e.g. from 20 to 120) for a few zones.

File: config/zones/US-FLA-FPC.yaml
Comment (mathilde-daugy) [reply config/zones/US-FLA-FPC.yaml:1]: will accept this because i think the value from 2020 was too high

**Pull request merged**